### PR TITLE
minor changes in stroopwafelinterface to work with runSubmit

### PR DESCRIPTION
**Pull request by LiekeVanSon**
Resolving issues raised in #741 .

Stroopwafel now runs without an error. MC only runs properly.
Full AIS needs to be tested more thoroughly

### CODE REVIEW

File: utils/preProcessing/stroopwafelInterface.py [hunk 1]
```diff
@@ -234,10 +234,11 @@ def rejected_systems(locations, dimensions):
     # Over-ride with pythonSubmit parameters, if desired
```
Comment (avigna) [inline utils/preProcessing/stroopwafelInterface.py]: Change this comment to:
Over-ride with runSubmit + compasConfigDefault parameters, if desired

Same in line 22 of this file, where the comment "Include options from local pythonSubmit file" should be "Include options from local runSubmit + compasConfigDefault files".

**Review by avigna — CHANGES_REQUESTED** [review]
Address changes in comments.

**Comment by avigna** [comment]
On the side: closing this issue *might* help with #753

**Review by avigna — APPROVED** [review]

**Comment by stellarphysicsandpopulations** [comment]
(Stephen) 
Lieke's fix worked great for me, except that I needed to pass the full path to the YAML file to runSubmit, even though it's the same directory as my runSubmit. (Edit: Using "!" for comment lines below, since hashes give weird formatting.)

!! programOptions = pythonProgramOptions()
!! the above doesn't work, somehow... I needed to pass the explicit location of the yaml file, so:
yaml_file_location = os.path.join(path_to_runSubmit, 'compasConfigDefault.yaml')
programOptions = pythonProgramOptions(config_file=yaml_file_location)

I don't know why. I suspect this is *not* because of the changes in the Stroopwafel version I'm using. I guess it's because I'm using the Docker version of COMPAS?  (Or maybe my installation is also somehow weird in a different way...)

(So far I'm hard-coding an assumption about the name of the file, and that it is in the same location as runSubmit.py, but that can be changed.)

(Tagging @FloorBroekgaarden too, in case this saves time for others.)

**Comment by stellarphysicsandpopulations** [comment]
(Stephen) 
Lieke's fix worked great for me, except that I needed to pass the full path to the YAML file to runSubmit, even though it's the same directory as my runSubmit:

##### programOptions = pythonProgramOptions()
##### the above doesn't work, somehow... I needed to pass the explicit location of the yaml file, so:
yaml_file_location = os.path.join(path_to_runSubmit, 'compasConfigDefault.yaml')
programOptions = pythonProgramOptions(config_file=yaml_file_location)

I don't know why. I suspect this is *not* because of the changes in the Stroopwafel version I'm using. I guess it's because I'm using the Docker version of COMPAS?  (Or maybe my installation is also somehow weird in a different way...)

(So far I'm hard-coding an assumption about the name of the file, and that it is in the same location as runSubmit.py, but that can be changed.)

(Tagging @FloorBroekgaarden too, in case this saves time for others.)

**Comment by stellarphysicsandpopulations** [comment]
Thinking more clearly, it can't be anything to do with me using the Docker version, at least not directly... this setup is well before we get to the Docker call.  Interesting.  Oh well, it was fun to investigate, whatever the reason.

**Comment by avigna** [comment]
What sort of error do you get, Stephen?

**Comment by stellarphysicsandpopulations** [comment]
With the fix above, no error.  If I don't pass the full path to the YAML file then runSubmit somehow doesn't find the YAML file (even though it's in the same directory).   

Not an issue for me, now.  Whether it's important or not for other depends on how weird my case it.

**Pull request merged**